lemon/market_data/api.py

```diff
@@ -27,6 +29,15 @@ def __init__(
         self._trades = Trades(self)
         self._quotes = Quotes(self)
         self._ohlc = Ohlc(self)
+        self._live_streaming = LiveStreaming(
```

Comment:
this object shouldn't be defined here. The user should be able to use live streaming in following way:

```
client = api.create(...)
client.streaming.authenticate()

# not
# client.market_data.live_streaming.authenticate()

```
We should have this object defined on the same level as market_data and trading.

Reply:
Sure can do. It is however market_data which we are streaming, we still dont support trading of bidirectional streams which could be expected

Reply:
live_streaming does not exist in market_data endpoints hierarchy (the MarketDataAPI object should represent the hierarchical structure of the md endpoints, as they are) - it is a separate product from SDK perspective.

Think about this SDK as tiny wrapper around some APIs. For each API you have separate object representing this API.

Reply:
updated